JustSaying.IntegrationTests/Fluent/IntegrationTestBase.cs

```diff
@@ -110,11 +112,14 @@ protected async Task RunActionWithTimeout(Func<CancellationToken, Task> action)
 
                 if (resultTask == delayTask)
                 {
-                    throw new TimeoutException($"The bus related action took longer to execute than the timeout of {Timeout.TotalSeconds} seconds");
+                    throw new TimeoutException($"The tested action took longer than the timeout of {Timeout} to complete.");
+                }
+                else
+                {
+                    cts.Cancel();
```

Comment:
Out of interest, is there a reason we have the else condition when we throw on the first one being true?

Reply:
I didn't go to the talk, so I don't actually know. I was just copying the sample. Before I opened the code I didn't realise it was 99% the same as what was in the sample already anyway.

Reply:
No braces? 🤢

Reply:
This is the hill I will die on 😆